src/modules/navigator/rtl_params.c

```diff
@@ -34,19 +34,20 @@
 /**
  * @file rtl_params.c
  *
- * Parameters for RTL
+ * Parameters for return to home mode.
```

Comment:
It is "Return mode" not "return to home mode". In particular this helps because rally points and mission landing patterns can change where you go.

Reply:
Lots of other cases.